dashboards/Insight/Prometheus_Exporter_Status.json

```diff
@@ -2058,13 +2170,18 @@
       "targets": [
         {
           "calculatedInterval": "10s",
+          "datasource": {
+            "type": "prometheus",
+            "uid": "PA58DA793C7250F1B"
+          },
```

Comment:
Can you change datasource to "datasource": "Metrics" ?

Reply:
Done